test/integration/crcsuite/crcsuite.go

```diff
@@ -308,6 +310,20 @@ func StartCRCWithDefaultBundleAndHypervisorSucceedsOrFails(hypervisor string, ex
 	return err
 }
 
+func StartCRCWithDefaultBundleAndNameServerSucceedsOrFails(nameserver string, expected string) error {
+
+	var cmd string
+	var extraBundleArgs string
+
+	if !constants.BundleEmbedded() {
+		extraBundleArgs = fmt.Sprintf("-b %s", bundleName)
+	}
```

Comment:
@tsedmik @jsliacan I would prefer to remove this logic in this PR then having it. Later once @jsliacan PR is in we should update it with that logic. Sounds good?